app/js/lib/foxbox-db.js

```diff
@@ -56,6 +56,17 @@ export default class FoxboxDb {
     }
   }
 
+  drop() {
```

Comment:
Naming is hard, but what about something like `clear`. That's consistent with that's in `foxbox-db.js`.